inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/PdfAnnoRenderer.java

```diff
@@ -157,11 +158,11 @@ private static Span convertToPdfAnnoSpan(VSpan aVSpan, String aColor, String aDo
             } catch (NoMatchFoundException e) {
                 // if no match is found stop search here. increasing context won't help
                 LOG.error("Could not find a match for existing annotation with id "
-                    + aVSpan.getVid().toString());
+                    + aVSpan.getVid().toString() + " and text \"" + annotatedText + "\"");
```

Comment:
When logging via SLF4J, use placeholders instead of String concatenation:

```
LOG.error("Could not find a match for existing annotation with id [{}] and text [{}]", aVSpan.getVid(), annotatedText);
```